### PR TITLE
Adds Bambora Gateway

### PR DESCRIPTION
**Pull request by molbrown**
Continues #2394 to include needed updates for support:
 - Renames to Bambora Asia-Pacific
 - removes Bambora Ready test files
 - includes `void` action

ECS-159

Unit:
7 tests, 47 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Remote:
11 tests, 20 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

File: lib/active_merchant/billing/gateways/bambora_apac.rb [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        end
+        xml.target!
+      end
+
+      def commit_url
+        (test? ? test_url : live_url)
+      end
+
+      def success_from(response)
+        (response[:response_code] == "0")
```
Comment (curiousepic) [inline lib/active_merchant/billing/gateways/bambora_apac.rb]: No need for parens here or in `commit_url` above

**Comment by curiousepic** [comment]
This looks good, though there are a bevy of rubocop violations to clean up.

**Comment by molbrown** [comment]
I went ahead and added `void` and fixed Rubocop (only remaining violation is in Fat Zebra).

**Comment by jknipp** [comment]
@molbrown The Fat Zebra violation is fixed in `master`.

**Pull request merged**